src/hb-ot-maxp-table.hh

```diff
@@ -39,6 +39,36 @@ namespace OT {
 
 #define HB_OT_TAG_maxp HB_TAG('m','a','x','p')
 
+struct maxpV1Tail
+{
+  inline bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
+
+  HBUINT16 maxPoints;             /* Maximum points in a non-composite glyph. */
+  HBUINT16 maxContours;           /* Maximum contours in a non-composite glyph. */
+  HBUINT16 maxCompositePoints;    /* Maximum points in a composite glyph. */
+  HBUINT16 maxCompositeContours;  /* Maximum contours in a composite glyph. */
+  HBUINT16 maxZones;              /* 1 if instructions do not use the twilight zone (Z0),
+                                     or 2 if instructions do use Z0; should be set to 2 in
```

Comment:
Please add '*' to the start of comment continuation lines.

Reply:
Done.